security/rbac/docs/index.rst

```diff
@@ -0,0 +1,460 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+General rolebinding syntax:
+
+.. code:: bash
+
+   confluent iam rolebinding create --role [role name] --principal User:[username] --resource [resource type]:[resource name] --[cluster type]-cluster-id [insert cluster id] 
+
+Available role types and permissions can be found :ref:`here <rbac-predefined-roles>`.
+
+Resource types include: Cluster, Group, Subject, Connector, TransactionalId, Topic.
+
+
+Listing a Users roles
+~~~~~~~~~~~~~~~~~~~~~
+
+General listing syntax:
+
+.. code:: bash
+
+   confluent iam rolebinding list User:[username] [clusters and resources you want to view their roles on]
+
+For example, list the roles of ``User:bender`` on Kafka cluster ``KAFKA_CLUSTER_ID``
+
+.. code:: bash
+
+   confluent iam rolebinding list --principal User:bender --kafka-cluster-id $KAFKA_CLUSTER_ID 
+
+
+.. _rbac_demo_docker:
+
+Docker
+======
+
+This method of running the demo is for users who have Docker.
+This demo setup includes:
+
+-  |zk|
+-  Kafka with MDS, connected to the OpenLDAP
+-  |sr|
+-  KSQL
+-  |kconnect-long|
+-  |crest|
+-  |c3|
+-  OpenLDAP
+
+Prerequisites
+-------------
+
+-  Docker
+-  ``zookeeper-shell`` must be on your ``PATH``
+-  :ref:`Confluent CLI <cli-install>`:
+   |confluent-cli| must be installed on your machine, version
+   ``v0.127.0`` or higher (note: as of CP 5.3, the |confluent-cli| is a separate
+   :ref:`download <cli-install>`
+
+
+Image Versions
+--------------
+
+-  You can use production or pre-production images. This is configured
+   via environment variables ``PREFIX`` and ``TAG``.
+
+   -  ``PREFIX`` is appended before the actual image name, before ``/``
+   -  ``TAG`` is a docker tag, appended after the ``:``
+   -  E.g. with ``PREFIX=confluentinc`` and ``TAG=5.3.1``, kafka will
+      use the following image: ``confluentinc/cp-server:5.3.1``
+   -  If these variables are not set in the shell, they will be read
+      from the ``.env`` file. Shell variables override whatever is set
+      in the ``.env`` file
+   -  You can also edit ``.env`` file directly
+   -  This means all images would use the same tag and prefix. If you
+      need to customize this behavior, edit the ``docker-compose.yml``
+      file
+
+Run the demo
+------------
+
+#. Clone the `confluentinc/examples <https://github.com/confluentinc/examples>`__ repository from GitHub and check out the :litwithvars:`|release|-post` branch.
+
+   .. codewithvars:: bash
+
+       git clone git@github.com:confluentinc/examples.git
+       cd examples
+       git checkout |release_post_branch|
+
+#. Navigate to ``security/rbac/scripts`` directory.
+
+   .. codewithvars:: bash
+
+       cd security/rbac/rbac-docker
+
+#. To start |cp|, run
+
+   .. code:: bash
+
+      ./confluent-start.sh
+
+You can optionally pass in where ``-p project-name`` to name the
+docker-compose project, otherwise it defaults to ``rbac``. You can use
+standard docker-compose commands like this listing all containers:
```

Comment:
the indentation is wrong with the code samples, e.g. the grey bar vertical bar should not be present:

![image](https://user-images.githubusercontent.com/11722533/68430831-b2203000-0165-11ea-8e02-07a10088b7fe.png)

Reply:
Ack